models/demos/falcon7b_common/tests/run_falcon_end_to_end.py

```diff
@@ -230,12 +230,12 @@ def run_test_FalconCausalLM_end_to_end(
                 use_cache=use_cache,
             )
         for device in devices:
-            tt_lib.device.Synchronize(device)
+            ttnn.synchronize_device(device)
         profiler.end("first_model_run_with_compile", force_enable=e2e_perf)
 
         # Dump device profiler data before second run to avoid exceeding profiler memory limits when using tracy
         for device in devices:
-            tt_lib.device.DumpDeviceProfiler(device)
+            ttnn.experimental.device.DumpDeviceProfiler(device)
```

Comment:
ttnn.experimental.device does not exist as far as I am aware, has this been tested?

Reply:
We changed this based on the tt-eager bindings, 
https://github.com/tenstorrent/tt-metal/blob/802e9a3edd63d5eec5837ae09367a03ff8c830a9/ttnn/cpp/ttnn/deprecated/tt_lib/csrc/tt_lib_bindings.cpp#L210 
and based on the inputs we get from TTNN team in this sheet here
https://docs.google.com/spreadsheets/d/1FKSFDpkM7iXPYQ8nMgjwLkBQK50J6kxFk25j4s-XaZA/edit?usp=sharing

Reply:
@KalaivaniMCW were you able to run the model pipelines I suggested above? If they pass then there is no issue. However, I tried `ttnn.experimental.device` earlier today and it did not exist so I would expect it to fail. @arakhmati could you confirm on this?

Reply:
@skhorasganiTT
As per @eyonland 's suggestion, I have imported the device module from tt_lib_bindings as `ttnn.experimental.device` 
and ran Model perf CI https://github.com/tenstorrent/tt-metal/actions/runs/10386170948. 
any other specific tests to track these changes?

Reply:
Great, thank you, that should be sufficient.